### PR TITLE
Update xcm pages

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/389/files
https://github.com/PureStake/moonbeam-docs/pull/414/files

Does not contain:

https://github.com/PureStake/moonbeam-docs/pull/398/files

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/xcm/fees.md [hunk 1]
```diff
@@ -73,73 +68,81 @@ Alice请求的资产转移过程如下：
...
-| :---------------: | :-------------------------------------------: | :--------------------------------------------: |
+|:-----------------:|:---------------------------------------------:|:----------------------------------------------:|
 | RocksDB (default) | {{ networks.polkadot.rocks_db.read_weight }}  | {{ networks.polkadot.rocks_db.write_weight }}  |
 |     ParityDB      | {{ networks.polkadot.parity_db.read_weight }} | {{ networks.polkadot.parity_db.write_weight }} |
 
 在指令权重花费的计算架构完成后，您能够以DOT为单位计算指令的花费。
 
-在波卡中，[`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/weights/extrinsic_weights.rs#L56){target=_blank}被设置为`{{ networks.polkadot.extrinsic_base_weight.display }}`，也就是一分的十分之一。一分为`10^10 / 10,000`。因此，在计算以DOT为单位的最终费用的公式中包含一个常量：
+在波卡中，[`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/weights/extrinsic_weights.rs#L56){target=_blank}被设置为`{{ networks.polkadot.extrinsic_base_weight.display }}`，也就是[一分的十分之一](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs#L87){targer=blank}。一分为`10^10 / 10,000`。
```
Comment (eshaben) [inline builders/xcm/fees.md:94]:
```suggestion
在波卡中，[`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/weights/extrinsic_weights.rs#L56){target=_blank}被设置为`{{ networks.polkadot.extrinsic_base_weight.display }}`，也就是[一分的十分之一](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs#L87){targer=blank}。一分为`10^10 / 100`。
```

File: builders/xcm/fees.md [hunk 2]
```diff
@@ -153,124 +156,129 @@ Kusama上的总权重花费包括：给定指令本身花费和数据库读写
...
 3. 在**MoonbeamRuntimeXcmConfigAssetType**选取**Xcm**
-
 4. 在**parents**一栏输入`1`
-
 5. 在**interior**选取`Here`
-
 6. 点击**+**提交查询
 
-xcDOT的`UnitsPerSecond`数值为`{{ networks.moonbeam.xcm.units_per_second.xcdot.display }}`
+xcDOT的`UnitsPerSecond`数值为`{{ networks.moonbeam.xcm.units_per_second.xcdot.display }}`。
```
Comment (eshaben) [inline builders/xcm/fees.md:277]:
```suggestion
xcDOT的`UnitsPerSecond`数值为`{{ networks.moonbeam.xcm.units_per_second.xcdot.transfer }}`。
```

File: builders/xcm/xcm-transactor.md [hunk 3]
```diff
@@ -5,90 +5,107 @@ description: 通过使用XCM-Transactor pallet，如何在其他链进行远程X
...
-XCM-transactor pallet包括以下只读存储方式：
+XCM-transactor pallet包含以下只读存储方法：
 
- - **destinationAssetFeePerSecond**() - 返回给定资产的每秒用时计算费用。这可以实现从重量到交易费用的转换
+ - **destinationAssetFeePerSecond**() — 返回给定multilocation资产的每秒费用。这能够将权重转换成费用。如果`feeAmount`设置为`None`，pallet extrinsicts将读取存储元素
  - **indexToAccount**(index) — 返回与给定衍生索引关联的原始链账户
- - **palletVersion**() — 从存储中返回当前pallet版本
- - **transactInfoWithWeightLimit**(location) — 返回给定multilocation的交易信息
+ - **palletVersion**() — 从存储库返回当前pallet的版本
+ - **transactInfoWithWeightLimit**(location) — 返回给定multilocation的交易信息。如果`feeAmount`设置为`None`，pallet extrinsicts将读取存储元素
```
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:86]:
```suggestion
 - **transactInfoWithWeightLimit**(location) — 返回给定multilocation的交易信息。如果`feeAmount`设置为`None`，pallet extrinsics将读取存储元素
```

File: builders/xcm/xcm-transactor.md [hunk 4]
```diff
@@ -119,74 +135,161 @@ XCM-transactor pallet包括以下用于获取pallet常量的只读函数：
...
 
-![XCM-transactor Transact Through Derivative Extrinsic](/images/builders/xcm/xcm-transactor/xcmtransactor-1.png)
+!!! 注意事项
 
-当交易完成后，您可以在[Moonbase Alpha](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/explorer/query/0x14cebfb2b4a4c0bf72cb2562344e6803263f45491d2ab14e7b91115ebd52e706){target=_blank}和[中继链](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/explorer/query/0x630456e6578d5b414db2e83486a6961ff90dc06bf979988b370dcb0e57550e41){target=_blank}中查看相关extrinsic和事件。请注意，在Moonbase Alpha中，有一个`transactThroughDerivative`相关联的事件，但是有一些`xcUNIT` Token已被销毁以偿还主权账户的交易费用。在中继链中，`paraInherent.enter` extrinsic会显示`balance.Transfer`事件，其中1 `UNIT` Token转移给Alice地址。尽管如此，交易费仍会通过Moonbase Alpha主权账户进行支付。
+`AssetsTrapped`事件在中继链上时因为XCM-transactor pallet尚不支持处理还款功能。因此，高出预估权重将会在目标链执行XCM时导致资产无法退回。
 
-## 获取注册的衍生索引
+### 获取已注册的衍生索引
```
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:166]:
```suggestion
### 获取已注册的衍生索引 {: #retrieve-registered-derivative-indexes }
```

File: builders/xcm/xcm-transactor.md [hunk 5]
```diff
@@ -119,74 +135,161 @@ XCM-transactor pallet包括以下用于获取pallet常量的只读函数：
...
+
+1. 选择您要发送XCM的账户，确保账户已[完成所有设置](#xcmtransactor-signed-check-prerequisites)
+
+2. 选择**xcmTransactor** pallet
+
+3. 选择**transactThroughSigned** extrinsic
+
+4. 定义目标multilocation。在本示例中，设置如下：
+
+    | Parameter |    Value    |
```
Comment (eshaben) [inline builders/xcm/xcm-transactor.md:215]: can these headers be translated?

**Comment by hyd628** [comment]
Thanks @eshaben , fixed.

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**